crypto/perlasm/x86_64-xlate.pl

```diff
@@ -688,6 +689,14 @@
 						      cfa_expression($$line)));
 				last;
 			      };
+	    /remember_state/
+			&& do {	push @cfa_stack, [$cfa_reg, $cfa_rsp];
+				last;
+			      };
+	    /restore_state/
+			&& do {	($cfa_reg, $cfa_rsp) = @{pop @cfa_stack};
+				last;
+			      };
```

Comment:
Should there be a check somewhere that the CFA stack is empty by the end of the perlasm script?

Reply:
I don't think there's a particular rule against unpaired remember_states, though I suppose it would be a little pointless.

Reply:
Yeah, it does sound like someone forgot something if they are unpaired, so a check would be helpful to the programmer

Reply:
Done.